apps/greenhouse/.gitignore

```diff
@@ -1 +1,2 @@
 public/extensions
+appProps.json
```

Comment:
Do we need again appProps.json? We have already a global entry for it:
https://github.com/cloudoperators/juno/blob/main/.gitignore#L8